discovery-provider/src/queries/notifications.py

```diff
@@ -589,6 +589,40 @@ def notifications():
                 }
                 created_notifications.append(track_notif)
 
+            # Tracks that were not remixes and turned into remixes
+            if not prev_entry.remix_of and entry.remix_of:
+                # Add notification to remix track owner
+                parent_remix_tracks = [t['parent_track_id'] for t in entry.remix_of['tracks']]
+                remix_track_parents = (
+                    session.query(Track.owner_id, Track.track_id)
+                        .filter(
+                            Track.track_id.in_(parent_remix_tracks),
+                            Track.is_unlisted == False,
+                            Track.is_delete == False,
+                            Track.is_current == True
+                        )
+                        .all()
+                )
+                for remix_track_parent in remix_track_parents:
+                    [remix_track_parent_owner, remix_track_parent_id] = remix_track_parent
+                    remix_notif = {
+                        const.notification_type: const.notification_type_remix_create,
+                        const.notification_blocknumber: entry.blocknumber,
+                        const.notification_timestamp: entry.created_at,
+                        const.notification_initiator: entry.owner_id,
+                        # TODO: is entity owner id necessary for tracks?
```

Comment:
i think this is one of maybe 2 notif types where entity id is actually the same

Reply:
Yeah -- I just copied the logic from the other. Probably should change both together (?). Will merge as is and follow up on it